### ISSUE TITLE
Server mix ups

### ISSUE THREAD

**Issue by mjb232** [jlm323/DecameronProject#41]
@mmm202  @laurenmcguigan  @jlm323 Hey, so I dont know if anyone else noticed but our server currently has another set of files in our server: there the usual stuff contained in our Decameron folder, and then theres a copy of our Website_files folder in there as well, which contains more of our files. Should I just delete the Website_Files folder?

**Comment by jlm323** [comment]
Yeah I think you can just delete the extra because our updated files aren't in that folder.

**Comment by mjb232** [comment]
gotcha

**Comment by mjb232** [comment]
Wait, no. I dont have permission to delete any of that stuff. Who ever put it there needs to delete it.

**Comment by jlm323** [comment]
I just tried to delete it and I don't have permission either.

**Comment by laurenmcguigan** [comment]
I deleted it, but the members page still isn't there @mmm202 @mjb232 @jlm323

**Comment by mmm202** [comment]
@mjb232 I still need ypur picture and info for the members page, I won't be able to do it until tomorrow evening because of my finals.

**Comment by mjb232** [comment]
Sorry, @mmm202. I'll add the summary and picture myself so don't worry about it
